generator/src/main/java/com/scottlogic/deg/generator/inputs/validation/messages/GranularityConstraintValidationMessages.java

```diff
@@ -0,0 +1,25 @@
+package com.scottlogic.deg.generator.inputs.validation.messages;
+
+import java.math.BigDecimal;
+
+public class GranularityConstraintValidationMessages implements StandardValidationMessages {
+
+
+    private BigDecimal validValue;
```

Comment:
These (and fields in other validation types) can be made `final`

Reply:
added where applicable.